vistrails/gui/uvcdat/graphicsMethodsWidgets.py

```diff
@@ -2058,11 +2059,18 @@ def initValues(self, gm=None):
         self.boxfillTypeButtonGroup.setChecked(gm.boxfill_type)
         self.clickedBoxType()
         
-        if gm.ext_1 == "n":
+        ## Apparently the first time around it comes in as expected with a boxfill gm
+        ## But right after that it comes back as a:
+        ## <class 'core.utils.InstanceObject'> with values no longer bool but str
+        ## So we need to test against this!
+        ## But we should really figure out why the type is changed to this vistrails thing
+        ## someone with a better understanding a VisTrails inners should try
+        ## to figure this out (@remram44 @vvpalav or @hgohil2805 ???
+        if gm.ext_1 in [False,0,"n", "False"]:
             self.ext1ButtonGroup.setChecked('No')
         else:
             self.ext1ButtonGroup.setChecked('Yes')
-        if gm.ext_2 == "n":
+        if gm.ext_2 in [False,0,"n","False"]:
```

Comment:
shoul just be False or True but to be safe I kept all valid "user input" in. the "False" is due to Vistrails it is not even a valid value but somehow when coming from Vistrails it has this value...